frontend/src/components/ResultRow.tsx

```diff
@@ -0,0 +1,127 @@
+import { StyleSheet, Text, View } from "react-native";
+import React from "react";
+import { BLUE, GREY, ORANGE } from "../constants/colors";
+interface ResultRowProps {
+    turkish: string,
+    englishSelectingWord?: string,
+    english?: string,
+    correctEnglish: string,
+    showingAnswers: boolean, 
+    isPairing: boolean
+}
+
+export default function ResultRow({ turkish, englishSelectingWord, english, showingAnswers, correctEnglish, isPairing }: ResultRowProps) {
+    const correct = english === correctEnglish;
+    const unusedSelectingWord = (!english) && !isPairing;
+    
+    let extraTurkishInfo;
+    let extraEnglishInfo;
+    let extraContainerInfo;
```

Comment:
Could we give these more descriptive names, possibly `additionalTurkishStyling` or something along those lines? Do let me know if I've misunderstood their purpose